### PR TITLE
User Can View All Grades

### PR DESCRIPTION
**Pull request by htjahyon**
image

### CODE REVIEW

**Comment by htjahyon** [comment]
image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]
@htjahyon This looks very different from the wireframe. Please try to match the wireframe as closely as possible:
---
Wireframe

**Comment by htjahyon** [comment]
image

File: index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Student Grade Table</title>
+    <link href="style.css" rel="stylesheet">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.3/css/bootstrap.min.css" rel="stylesheet">
```
Comment (TimelikeClosure) [inline index.html:6]: Always include your custom stylesheets after library stylesheets to take advantage of cascading.

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**